### ISSUE TITLE
Scrolling issue Brendor's intro dungeon.

### ISSUE THREAD

**Issue by Tricky1975** [Tricky1975/Old-Games#1]
(Reported by jtrev, copied here)

https://drive.google.com/file/d/0B5APKZT75uLbdm1ocmdydEszekE/view?usp=sharing

It's very likely an error in the coordinate settings of the x-axis-fixation. 
Shouldn't be hard to fix, but will have to wait until tomorrow (that is dec. 26th) as it's Christmas for me too. :santa:

**Comment by Tricky1975** [comment]
The data I found in my files render the possibility this can happen completely impossible, and when a reason for a bug doesn't exist, a way to fix it also doesn't exist.

Therefore I did not see any other alternative than to resort to a "dirty" method that breaks down the game a bit in cosmetic way, but at least this bug can no longer happen at all (as I removed the block on horizontal scrolling, which was not the cause of the bug (as all that could cause this is a wrong x-asis positioning, but that is set to 1 or 0 as it should be), but if the bug happens now, the game now has a chance to "auto-correct" it, and thus fixing the bug by itsel.

I know I planned this for tomorrow, but as Christmas day is always a day in which nothing happens for many hours, I was bored. I will try to get the new fixed version up asap, though I cannot yet make any promises yet, as the day proceeds I may get busy too today, but I should be able to get the new version on prior to 2016 ;)
